tavern/internal/graphql/schema.graphql

```diff
@@ -1760,6 +1760,16 @@ input SubmitTaskResultInput {
   """Error message captured as the result of task execution failure."""
   error: String
 }
+input ImportTomesFromGitInput {
+  """Specify a git URL to obtain the tomes from."""
+  gitURL: String!
+
+  """
+  Optionally, specify directories to include.
+  Only tomes that have a main.eldritch in one of these directory prefixes will be included.
+  """
+  includeDirs: [String!]
```

Comment:
Does this search recursizely? Eg do i need to specify:
gitURL: https://github.com/spellshift/realm
includeDirs: ["tavern/tomes/"]
- or -
includeDirs: ["tavern/tomes/cat","/tavern/tomes/hostname",...]

Reply:
Just `tavern/tomes/`, all it does is a prefix match